pkg/block/indexheader/lazy_binary_reader.go

```diff
@@ -99,30 +98,34 @@ func NewLazyBinaryReader(
 	metrics *LazyBinaryReaderMetrics,
 	onClosed func(*LazyBinaryReader),
 ) (*LazyBinaryReader, error) {
-	filepath := filepath.Join(dir, id.String(), block.IndexHeaderFilename)
+	indexHeaderFile := ""
```

Comment:
Nit: I would prefer `var` statement if we're initializing new empty variable, it makes it more obvious to me.

Reply:
Done